lapce-rpc/Cargo.toml

```diff
@@ -10,7 +10,7 @@ edition = "2021"
 log = "0.4.17"
 parking_lot = "0.11.2"
 anyhow = "1.0.34"
-indexmap = "1"
+indexmap = { version = "1.9.2", features = ["serde"] }
```

Comment:
Added `features = ["serde"]` because it doesn't compile for testing otherwise (#[derive(Serialize, Deserialize)] doesn't work on IndexMap).